test/junit/scala/reflect/internal/InferTest.scala

```diff
@@ -36,28 +31,28 @@ class InferTest extends BytecodeTesting {
       val dsym = symbolOf[D[_]]
 
       val tp0 = typeOf[D[A]]
-      assert(tp0.contains(asym))
+      assertTrue(tp0.contains(asym))
```

Comment:
picking up the symboltable assert (from old refactoring) induces deprecation for one-arg form. Normally don't care assertTrue vs Predef.assert.